### PR TITLE
Add test & update

### PR DESCRIPTION
**Pull request by Patfarmurs**

### CODE REVIEW

File: spec/requests/posts_spec.rb
Comment (fobadara) [inline spec/requests/posts_spec.rb:1]: [REQUIRED] When I run your test I get some errors. Kindly fix them:

image

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Patfarmurs, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good use of describe block
✅ Use of context

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb
Comment (Patfarmurs) [reply spec/requests/posts_spec.rb:1]: Hi @fobadara, all tests are passing on my side.

Screenshot-exemple

**Review by BregornOriginal — APPROVED** [review]
Hi @Patfarmurs 

Your project is complete!  There is nothing else to say other than... it's time to merge it

Congratulations! 



## Highlights

✅ Linters are passing
✅ Good commit history
✅ Tests required are passing

### [OPTIONAL] ***Last reviewer was checking all your tests, that's why he find some failures. For this part of the project, you can continue as all the requirements are accomplished, but it would be great to fix your text suite as the previous reviewer mentioned. It's on files `spec/models/like_spec.rb` and `spec/models/comment_spec.rb`***

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Comment by Patfarmurs** [comment]
Thank you very much @BregornOriginal. 
I'll definitely tag you.

**Pull request merged**